
syntax = "proto3";

enum FollowType {
  FOLLOW_TYPE_INVALID = 0;             // 非法关注链类型
  FOLLOW_TYPE_FOLLOW = 1;              // 关注
  FOLLOW_TYPE_UNFOLLOW = 2;            // 取消关注
  FOLLOW_TYPE_BLOCK = 3;               // 屏蔽
  FOLLOW_TYPE_UNBLOCK = 4;             // 取消屏蔽
}

message FollowAction {
  FollowType type = 1;
  int64 time = 2;
  string user = 3;
  string target_user = 4;
}

